src/JoeScan.Pinchot.Shared/Point2D.cs

```diff
@@ -20,19 +20,19 @@ public struct Point2D : IEquatable<Point2D>
         /// Gets or sets the X coordinate value of the <see cref="Point2D"/> in inches.
         /// </summary>
         /// <value>The X coordinate value of the <see cref="Point2D"/> in inches.</value>
-        public double X { get; set; }
+        public double X { get; set; } = double.NaN;
 
         /// <summary>
         /// Gets or sets the Y coordinate value of the <see cref="Point2D"/> in inches.
         /// </summary>
         /// <value>The Y coordinate value of the <see cref="Point2D"/> in inches.</value>
-        public double Y { get; set; }
+        public double Y { get; set; } = double.NaN;
 
         /// <summary>
         /// Gets or sets the brightness value of the <see cref="Point2D"/>.
         /// </summary>
         /// <value>The brightness value of the <see cref="Point2D"/>.</value>
-        public int Brightness { get; set; }
+        public int Brightness { get; set; } = Globals.ProfileDataInvalidBrightness;
```

Comment:
@aaronhudon-ts Non-zero defaults for `struct`s are only valid for C# 10.0 (`net6.0` and above) see the [C# 10 page](https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-10#improvements-of-structure-types).

The v13 Pinchot API only supports up to `net5.0` so I would have to make a `net6.0` package. Is the application that consumes this API `net6.0`+?

Reply:
Yes, .net6.0
... Or you could have some compiler directives for >= NET6.0
... Or create a constructor in the `Point2D` that initializes the fields

Reply:
I have ideas on how to implement it. Just making sure I'm building something compatible for your system.